packages/pn-personafisica-webapp/package.json

```diff
@@ -76,6 +76,7 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^28.0.0",
     "@types/mixpanel-browser": "^2.38.0",
+    "@types/react-helmet": "^6.1.6",
```

Comment:
@LorenzoCocciarelli is helmet still used? if not remove it

Reply:
removed both react-helmet and @types/react-helmet